examples/simple.yml

```diff
@@ -0,0 +1,82 @@
+name: random python project name
+version: '0.1.4'
+dependencies:
+  platforms:
+    ubuntu:
+      versions:
+        - '14.04'
+        - '>16.04'
+      apt:
+        sharable:
+          - build-essential
+          - cmake
+          - python2
+          - curl
+    mac:
+      versions:
+        - '>=10.14'
+      brew:
+        sharable:
+          - cmake
+          - python@2
+          - curl
+    windows:
+      versions:
+        - '>=7600'
+      arch: x86_64
+      download_directory: 'C:\opt\Downloads'
+      download:
+        - uri: https://www.7-zip.org/a/7z1900-x64.msi
+          sha512: 7837a8677a01eed9c3309923f7084bc864063ba214ee169882c5b04a7a8b198ed052c15e981860d9d7952c98f459a4fab87a72fd78e7d0303004dcb86f4324c8
+          sharable: true
+        - uri: https://github.com/mridgers/clink/releases/download/1.0.0a1/clink-1.0.0a1.823d84.zip
+          sha512: 343f87eb67177d7bcb9982b34bceb2f056e12aa1140e1d12b25863ec033ad2f40735647cbd79cc528d8bddcd929df99546a7c398f79365f9cf246011c63f724f
```

Comment:
There are two forms for hashing shown in this commit. We should pick one. I'm thinking this approach is better than having  `hash_protocol`. Would you agree?

Reply:
yes, it makes it easier